<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1165</width>
    <height>868</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #404040;
    color: #e6e6e6;
    margin: 0px;
    padding: 4px;
    border-radius: 4px;
}

QPushButton::checked {
     background-color: #5680c2;
     color:#fcfdfd;
}

QCheckBox {
    background: transparent;
    color: #e4e4d4;
}
QCheckBox::indicator {
    width: 14px;
    height: 15px;
}
QCheckBox::indicator:checked {
    image: url(images:QCheckBox_checked.png);
}
QCheckBox::indicator:checked:hover {
    image: url(images:QCheckBox_checked_hover.png);
}
QCheckBox::indicator:unchecked {
    image: url(images:QCheckBox_unchecked.png);
}
QCheckBox::indicator:unchecked:hover {
    color: #f5f5f5;
    image: url(images:QCheckBox_unchecked_hover.png);
}


QComboBox {
    background-color: #2c2c2c;
    border: 1px solid #404040;
    color: #d0d0d0;
    padding-left: 6px;
}
QComboBox:disabled {
    background-color: #323232;
    color: #8a8a8a;
}
QComboBox::down-arrow {
    background-color: #2c2c2c;
    image: url(images:QComboBox_down_arrow.png);
}
QComboBox::down-arrow:on {
    background-color: #5680c2;
}
QComboBox::drop-down {
    background-color: #202020;
}
QComboBox::on {
    background-color: #5680c2;
    color: #f8f9f9;
}
QComboBox QAbstractItemView {
    border-bottom-left-radius: 4px;
    border-bottom-right-radius: 4px;
    selection-background-color: #5680c2;
}

QScrollBar{
    margin: 0px;
    padding: 0px;
}

QScrollBar:vertical {
    border: 1px solid #3D3A3A;
    background-color: none;
    width: 8px;
    margin: 0px 0px 0px 0px;
}

QScrollBar:horizontal {
    border: 1px solid #3D3A3A;
    background-color: none;
    height: 8px;
    margin: 0px 0px 0px 0px;
}

QScrollBar::handle:vertical {
    background-color: #545454;
    min-height: 1px;
    border-radius: 4px;

}
QScrollBar::handle:horizontal {
    background-color: #545454;
    min-width: 1px;
    border-radius: 4px;
}

QScrollBar::add-line:vertical {
    background-color: RED;
    height: 0px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical {
    background-color: RED;
    height: 0px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}

QScrollBar::add-line:horizontal {
    background-color: RED;
    width: 0px;
    subcontrol-position: right;
    subcontrol-origin: margin;
}

QScrollBar::sub-line:horizontal {
    background-color: RED;
    width: 0px;
    subcontrol-position: left;
    subcontrol-origin: margin;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
    background-color: #202020;
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
    background-color: #202020;
}

QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
    background-color: RED;
    height: 0px;
    width: 0px;
}

QScrollBar::left-arrow:horizontal, QScrollBar::right-arrow:horizontal {
    background-color: RED;
    height: 0px;
    width: 0px;
}


QGroupBox {
    border: 0px solid #373737;
    background-color: #373737;
    padding-top: 12px;
}
QGroupBox::title {
    background-color: transparent;
    margin-top: 4px;
    padding-left: 8px;
}


QHeaderView {
    background-color: #3b3b3b;
    border: 0px transparent transparent;
}
QHeaderView:section {
    background-color: #3b3b3b;
    border: 0px transparent transparent;
}


QLabel {
    background-color: transparent;
    color:#e6e6e6
}


QLineEdit {
    background-color: #2c2c2c;
    border: 1px solid #404040;
    color: #dcdcdc;
}


QListView {
    background-color: #232323;
    border: 0px transparent transparent;
}
QListView:disabled {
    background-color: #19232D;
    color: #787878;
}
QListView:item {
    background-color: #282828;
}
QListView:item:hover {
    background-color: #484848;
    border: 0px transparent transparent;
}
QListView:item:selected {
    background-color: #314e78;
    border: 0px transparent transparent;
    color: #c68652;
}


QProgressBar {
    background-color: #424242;
    border: 1px solid #373737;
    padding: 0px;
    text-align: right;
}
QProgressBar::chunk {
    background-color: #5680c2;
    border: 1px solid #373737;
}

QMessageBox {
    background-color: rgb(24, 24, 24);
}

QMessageBox QPushButton {
    min-width: 105px;
}

QPushButton {
    background-color: #585858;
    border: 1px solid #404040;
}
QPushButton:default {
    background-color: #5379b4;
}
QPushButton:disabled {
    background-color: #323232;
    color: #8a8a8a;
}
QPushButton:hover {
     background-color: #676767;
}
QPushButton:pressed {
     background-color: #5680c2;
     color:#fcfdfd;
}

QRadioButton {
    background: transparent;
    color: #e4e4d4;
}
QRadioButton::indicator {
    width: 14px;
    height: 14px;
}
QRadioButton::indicator:checked {
    image: url(images:QRadioButton_checked.png);
}
QRadioButton::indicator:checked:hover {
    image: url(images:QRadioButton_checked_hover.png);
}
QRadioButton::indicator:unchecked {
    image: url(images:QRadioButton_unchecked.png);
}
QRadioButton::indicator:unchecked:hover {
    color: #f5f5f5;
    image: url(images:QRadioButton_unchecked_hover.png);
}


QSpinBox {
    background: #595959;
    border: 1px solid #404040;
    text-align: center;
}
QSpinBox::down-button {
    border-width: 0px;
    padding-top: 3px;
    subcontrol-origin: control;
    subcontrol-position: top left;
    width: 13px;
    height: 20px;
}

QSpinBox::down-button:hover {
    background:#6b6b6b
}
QSpinBox::down-arrow:hover {
    height: 20px;
    image: url(images:QSpinBox_down_arrow.png);
}
QSpinBox::hover {
    background: #808080;
}
QSpinBox::up-button {
    border-width: 0px;
    padding-top: 3px;
    subcontrol-origin: control;
    subcontrol-position: top right;
    width: 13px;
    height: 20px;
}
QSpinBox::up-button:hover {
    background:#6b6b6b
}
QSpinBox::up-arrow:hover {
    height: 20px;
    image: url(images:QSpinBox_up_arrow.png);
}


QTableView QTableCornerButton:section {
    background-color: #3b3b3b;
    border: 1px solid #3b3b3b;
}


QTabWidget::pane {
    background-color: #404040;
    border-color: #373737;
    border-style: solid;
    border-width: 2px;
    border-radius: 4px;
    margin-left: 1x;
}
QTabBar::tab {
    background-color: #373737;
    border-style: solid;
    border-width: 1px;
    border-top-color: #373737;
    border-right-color: #373737;
    border-left-color: #373737;
    border-bottom-color: #373737;
    padding: 5px;

}
QTabBar::tab:selected {
    background-color: #424242;
    border-style: solid;
    border-width: 1px;
    border-color: #373737;
    border-bottom-color: #424242;
}
QTabBar::tab:hover:!selected {
    background-color: #343434;
    color: #afafaf;
}
QTabBar::tab:!selected {
    border-style: solid;
    border-width: 1px;
    background-color: #313131;
    color: #d8d8d8;
}


QTreeView {
    background-color: #232323;
    border: 0px transparent transparent;
}
QTreeView:disabled {
    background-color: #19232D;
    color: #787878;
}
QTreeView:item {
    background-color: #282828;
}
QTreeView:item:hover {
    background-color: #484848;
    border: 0px transparent transparent;
}
QTreeView:item:selected {
    background-color: #314e78;
    border: 0px transparent transparent;
    color: #c68652;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QListWidget" name="listWidget"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>BLENDER EXECUTABLE : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="b_exec">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="render_button">
         <property name="text">
          <string>START RENDER</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="labelAlignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>CURRENT TASK :</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QProgressBar" name="pbc">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>TOTAL :</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QProgressBar" name="pbt">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item alignment="Qt::AlignHCenter|Qt::AlignTop">
        <widget class="QPushButton" name="plusb">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="minb">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>25</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="logbutton">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="horizontalWidget_2" native="true">
     <layout class="QHBoxLayout" name="log_layout">
      <item>
       <widget class="QTextEdit" name="logtext">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="placeholderText">
         <string>no logs</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>25</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>logbutton</sender>
   <signal>toggled(bool)</signal>
   <receiver>horizontalWidget_2</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1146</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>1149</x>
     <y>443</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
